proto/scene.proto

```diff
@@ -81,6 +89,14 @@ message MeshedObject {
 }
 
 // Representation of object to render
+// A sceneObject is one of the main things of this format. It describes all the required fields to use if you want to render soemthing.
+// SceneObject contains:
+// - id: defines the id for whole scene.
+// - transform: this part has all data for transformation of the object.
+// - material: this part contains information about the material, which should be applied to object.
+// - mesh: this is the object itself. It may be Plane, Sphere or link to am .obj file, which you are already able to parse.
```

Comment:
`an`.
`It may be Plane, Sphere or link to am .obj file` - maybe remove this? It can be a bit confusing because there are more options available.